src/channel.js

```diff
@@ -241,9 +241,10 @@ export class Channel {
 		return data;
 	}
 
-	async addMembers(members) {
+	async addMembers(members, message) {
```

Comment:
Would be nice to have short function doc -`members` param is obvious, but short explanation about message will be helpful I think!!

Something similar to:

<img width="928" alt="Screen Shot 2019-11-05 at 11 12 23 AM" src="https://user-images.githubusercontent.com/11586388/68198921-469b5e80-ffbd-11e9-908a-90dccbf5746c.png">